backend/src/packages/articles/libs/helpers/get-article-pubished-status-query.helper.ts

```diff
@@ -0,0 +1,18 @@
+import { type QueryBuilder } from 'objection';
+
+import { type ArticleModel } from '../../article.model.js';
+import { type ArticleGenreStatsFilters } from '../types/types.js';
+
+const getArticlePublishedStatusQuery = (
+  articleStatus: ArticleGenreStatsFilters['articlePublishedStatus'],
+): ((builder: QueryBuilder<ArticleModel, ArticleModel[]>) => void) => {
+  return (builder: QueryBuilder<ArticleModel, ArticleModel[]>): void => {
+    if (articleStatus) {
+      articleStatus === 'draft'
```

Comment:
Let's create `ArticlePublishStatus` enum in shared and use it on backend\frontend
```suggestion
      articleStatus === ArticlePublishStatus.DRAFT
```